src/modules/images/ImagePropertiesModal.js

```diff
@@ -117,9 +119,11 @@ const ImagePropertiesModal = (props) => {
     if (isEmpty(annotated) && props.selectedImage.hasOwnProperty('annotated')) delete props.selectedImage.annotated;
     else if (!isEmpty(annotated)) props.selectedImage.annotated = annotated;
     console.log('Saving form data to Spot ...', props.selectedImage);
-    let images = props.spot.properties.images;
+    let spot = useSpots.getSpotByImageId(props.selectedImage.id);
+    let images = JSON.parse(JSON.stringify(spot.properties.images));
```

Comment:
If we use `JSON.parse(JSON.stringify` to make a copy we try to put the word Copy in the variable name so this should be `imagesCopy`